parseRecipes.js

```diff
@@ -9,9 +9,19 @@ var difficulties = {
   'Beginner': 1
 };
 
-var categoryOrder = ['General Usage', 'Offline', 'Beyond Offline', 'Performance', 'Web Push'];
+var categories = module.exports.categories = [
+  { title: 'General Usage', slug: 'general-usage' },
+  { title: 'Offline', slug: 'offline' },
+  { title: 'Beyond Offline', slug: 'beyond-offline' },
+  { title: 'Performance', slug: 'performance' },
+  { title: 'Web Push', slug: 'web-push' }
+];
 
-module.exports = function(recipeSlugs) { 
+var categoryOrder = categories.map(function(category) {
+  return category.title;
+});
```

Comment:
Could we order them as before? (simple to hard, then `performance` and then `web push`)